extensions/training-data/README.md

```diff
@@ -0,0 +1,55 @@
+
+- [Example Item](example-roads.json)
+- [Example Asset (labels)](example-labels.json)
+- [Example Source Imagery](example-source.json)
+- [JSON Schema](schema.json)
+
+A Training Data Item represents a set of polygons with labels and, like other STAC Items should be part of a Collection. It is up to the data provider how to group their catalog, but a typical use might have a Collection of a series of training data sets (Items) that are related. For example a "Building" collection might have 50 Items, each one was a set of building AOIs for a single country. The Collection holds details on the data providers and the license.
+
+## Item fields
+Like other content extensions, the TD extension adds additional fields to a STAC Item, which are detailed after some additional clarification on what the core fields mean with respect to a TD Item.
+
+### Core Item fields
+Some additional notes are given here for some of the core STAC Item fields and what they represent for training data.
+
+- **bbox** and **geometry**: The bounding box and the geometry of a TD Item represents the region for which the training data is valid. This could be the extent of all the AOIs in the dataset, or could be the region the provider believes the training data is representative.
+- **properties.datetime**: The datetime of a TD Item is the nominal datetime for which the training data applies, typically this is the datetime of the source imagery used to generat the labels. If the training data applies over a range of datetimes (e.g., generated from multiple source images) then use the datetime-range (dtr) extension to indicate start and end datetimes.
+- **properties.assets**: Unlike other extensions, the TD extension requires a single asset with the key "labels". This is a GeoJSON FeatureCollection asset containing the actual label features. As with the core STAC Item a thumbnail asset is also strongly encouraged.
+
+### New Item properties
+| element         | type info       | name                       | description       | 
+|-----------------|-----------------|----------------------------|--------------------------------------------------------------------------------------------------| 
+| td:title | string | Title | A human readable title of the dataset for display |
+| td:description | string | Description | **REQUIRED.** A description of the training data, how it was created, and what it is recommended for |
+| td:classes | [string] | Classes | **REQUIRED.** a list of keywords representing the nature of the labels. (e.g., tree, building, car, hippo)
```

Comment:
Have you thought about supporting non-classificaiton tasks? e.g. "estimate count of  buildings in this 500x500 pixel grid". Requiring classes here wouldn't fit with regression problems. An easy work around would be to just give a single "value" class, but you may want to bake a flexibility in the data model instead.

Reply:
It's a good idea @lossyrob , so we could: 

- have a "value" property in the asset features rather than a "label" property
- td:classes renamed to td:values which could contains the list of possible values. Although for something like 'count' this would be unwieldy.

How would you define the task?  Right now we have this "label_type", which is: "One of 'classification', 'detection', or 'segmentation'", but I agree this is unnecessarily limiting. Going to write some more thoughts below.